grpc/grpc-macros/Cargo.toml

```diff
@@ -5,7 +5,7 @@ edition = "2021"
 
```

Comment:
could you please extend package metadata with missing stuff?
https://github.com/eigerco/lumina/actions/runs/12119063228/job/33784860143#step:6:447
> manifest has no description, license, license-file, documentation, homepage or repository.

Reply:
Added license, description, authors, homepage, repository, and readme keys :+1: